.github/workflows/tests.yml

```diff
@@ -4,9 +4,17 @@ on:
   push:
     branches:
       - dev-3.x
+    paths:
+      - "aiogram/**"
+      - "tests/**"
+      - ".github/workflows/tests.yml"
```

Comment:
`poetry.lock`, `pyproject.toml`, `.flake8`, `mypy.ini`, `codecov.yaml`, `.coveragerc` should also be added, as we need to check the code after updating the dependencies or configs